.github/workflows/ci.yml

```diff
@@ -46,14 +46,19 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Install libs
-        run: sudo apt-get install -y dbus dbus-x11 python-gi
```

Comment:
This should probably just be `python3-gi`. `python-gi` would install the Python 2 version.

Reply:
Let's give that a shot 👍

Reply:
Looks like they still get skipped since its not for the right python version